src/webgpu/util/copy_to_texture.ts

```diff
@@ -81,7 +109,9 @@ export class CopyToTextureUtils extends GPUTest {
     format: RegularTextureFormat,
     srcPremultiplied: boolean,
     dstPremultiplied: boolean,
-    isFlipY: boolean
+    isFlipY: boolean,
+    srcColorSpace: 'display-p3' | 'srgb' = 'srgb',
+    dstColorSpace: 'srgb' = 'srgb'
```

Comment:
I think this would more precisely be:
```suggestion
    dstColorSpace: GPUPredefinedColorSpace = 'srgb'
```

Reply:
Unfortunately, current GPUPredefinedColorSpace only have 'srgb', add a TODO for this.

Reply:
Right, which is exactly why it makes sense to use it here. Then when we update GPUPredefinedColorSpace this will get updated automatically.